### DISCUSSION TITLE
Error: Requires write access to '/src/blocklists__', but the file system on Deno Deploy is read-only

### DISCUSSION THREAD

**Discussion by aminkokabi** [serverless-dns/serverless-dns#176] [General]
it gives error like picture, how to fix it? thank you
![Untitled-1](https://github.com/serverless-dns/serverless-dns/assets/12424117/5e639f56-5149-4518-a1a0-aa4268b754e7)

**Comment by ignoramous** [comment]
On deno deploy, blocklists.ts should abort saving these files:
https://github.com/serverless-dns/serverless-dns/blob/cf0341dbddad0cd1eb9f128f10b5d0c1e9481eb1/src/core/deno/blocklists.ts#L17-L18

Not sure how this check fails for you (it does not, for me): 
https://github.com/serverless-dns/serverless-dns/blob/cf0341dbddad0cd1eb9f128f10b5d0c1e9481eb1/src/commons/envutil.js#L37-L46

Strange.

Are you overriding env var `CLOUD_PLATFORM`, by any chance?
